### PR TITLE
docs: Add extra instructions for GCE samples.

### PR DESCRIPTION
**Pull request by jasonborg**

### CODE REVIEW

File: samples/common_samples_prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+1. Give your project a name.
+1. Make a note of the project ID, which might be different from the project
+   name. The project ID is used in commands and in configurations.
+
+## Install the Snapshot Debugger CLI and enable Firebase
+
+Follow the instructions beginning at [Before you
+begin](../../../README.md#before-you-begin) through to and including [Enable
+Firebase for your Google Cloud
+Project](../../../README.md#enable-firebase-for-your-google-cloud-project) to
```
Comment (mctavish) [inline samples/common_samples_prerequisites.md:20]: I don't think that these links are correct.  Just one directory down?

File: samples/gce_samples_prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    gcloud iam service-accounts list
+    ```
+
+    Identify the email of the entry with a display name resembling `Compute
+    Engine default service account`.
+
+* **Add the role**
+
+    ```
+    gcloud projects add-iam-policy-binding jcb-gce-fresh-test \
```
Comment (mctavish) [inline samples/gce_samples_prerequisites.md:47]: Replace jcb-gce-fresh-test with [YOUR PROJECT ID]?

**Review by mctavish — APPROVED** [review]

File: samples/common_samples_prerequisites.md (hunk 1 above)
Comment (jasonborg) [reply samples/common_samples_prerequisites.md]: Thanks!

File: samples/gce_samples_prerequisites.md (hunk 2 above)
Comment (jasonborg) [reply samples/gce_samples_prerequisites.md]: THanks for catching!

**Pull request merged**